pkg/getter/httpgetter.go

```diff
@@ -53,6 +53,8 @@ func (g *HTTPGetter) get(href string) (*bytes.Buffer, error) {
 		return nil, err
 	}
 
+	req.Header.Set("Accept", "application/gzip,application/octet-stream")
```

Comment:
The `get`/`Get` function is used to download more than the archives. It's used for all downloading. For example, it retrieves the index files. I think we need more nuance in figuring out the right accept header to set.

Reply:
[Added some nuance](https://github.com/helm/helm/pull/13034)